src/figures/figureB1.jl

```diff
@@ -52,18 +52,18 @@ function PlotSynGraph()
     for (ii, value) in enumerate(IC)
         A = synergyGrid(4, value, FcgR, Kav)
         h = collect(Iterators.flatten(A))
-        S[ii, 1:4] = h[1:4]
-        S[ii, 5:7] = h[6:8]
-        S[ii, 8:9] = h[11:12]
-        S[ii, 10] = h[16]
+        S[ii, 1:5] = h[2:6]
+        S[ii, 5:8] = h[8:11]
+        S[ii, 8:9] = h[14:15]
+        S[ii, 10] = h[20]
```

Comment:
So here.

Reply:
Ok, I resolved this issue. It still won't build though.